hazelcast/src/main/java/com/hazelcast/internal/nio/tcp/TcpIpConnection.java

```diff
@@ -71,7 +71,8 @@
 
     private TcpIpConnectionErrorHandler errorHandler;
 
-    private volatile ConnectionType type = NONE;
+    @Probe
+    private volatile String connectionType = NONE;
```

Comment:
Do we support `String` probes? I don't see any support for this type in `FieldProbe` - it only supports something that can be represented as `long` or `double`.

Reply:
Only long/double probes

Reply:
Thanks for the info @pveentjer!

Reply:
It seems we don't. I don't like the idea of having a map just for this. We are investigating if supporting String is possible. But I will probably go with a `Map` for this pr.